src/uw_basic.jl

```diff
@@ -124,14 +124,146 @@ Compute resonance frequency of a freely oscillating has bubble in water, given:
+      Cp = 1.00e3;        # specific heat capacity of air in J Kg^-1 K^-1
+      γ = 1.40;       # ratio of specific heats
+    end
+
+    # Compute bubble gas density from the internal, equilibrium pressure
+    ρg::Float64 = M*pineq/(Rg*T);
+    # Compute thermal diffusivity of gas. Note Prosperetti (1976a, p.19) has a
+    # typo: Cv,g should be Cp,g as used beloω.
+    Dg::Float64 = Kg/(ρg*Cp);
+    ## Losses versus frequency for fixed radius and depth.
+    G1::Float64 = M*Dg*ω/(γ*Rg*T); #pg 19
+    G2::Float64 = ω.*(radius.^2)./Dg; #pg 19. essentialy the square of the ratio between the bubble tabus and the thermal penetration depth
+    G3::Float64 = ω.*(radius.^2)./Dw; #pg 19
+    F::Number = 1 .+ (1+1im)*.√(0.5*G3);
+    kratio::Float64 = Kw/Kg;
+
+    β1::Number = .√(0.5γ*G2.* (1im .- G1 .+ .√((1im .- G1).^2 .+ 4im*G1/γ)));
+    β2::Number = .√(0.5γ*G2.* (1im .- G1 .- .√((1im .- G1).^2 .+ 4im*G1/γ)));
+    λ1::Number = β1.*coth.(β1) .- 1;
+    λ2::Number = β2.*coth.(β2) .- 1;
+    Γ1::Number = 1im .+ G1 .+ .√((1im .- G1).^2 .+ 4im*G1/γ);
+    Γ2::Number = 1im .+ G1 .- .√((1im .- G1).^2 .+ 4im*G1/γ);
+    ψ::Number = (kratio*F.*(Γ2 .- Γ1) .+ λ2.*Γ2 .- λ1.*Γ1)./(kratio.*F.*(λ2.*Γ1 .- λ1.*Γ2)- λ1.*λ2.*(Γ2.-Γ1));
+
+    μth::Float64 = 1/4*(ω.*ρg.*radius.^2).*imag.(ψ);#effective thermal viscosity.
+    κ::Float64 = 1/3*((ω.^2).*ρg.*radius.^2 ./pineq).*real.(ψ); #polytropic exponent.
+    b_th::Float64 = 2μth./(ρ.*radius.^2); #thermal damping
+    b_ac::Float64 = 1/2 .* ω.* (ω.*radius/c)./(1 .+ (ω.*radius/c).^2); #acoustic damping
+    # b_vs = repeat([2μ./(ρ*radius.^2)], outer = [1,length(ω)]); #viscous damping
+    b_vs::Float64 = 2μ./(ρ.*radius.^2); #viscous damping
+    ω0::Float64 = .√(3κ .* pineq./(ρ.*radius.^2) .- 2σ./(ρ.*(radius.^3)))# + (k.^2).*(radius.^2)./(1 .+ (k.^2).*(radius.^2)).*(ω.^2) );
+    return [ω0,b_th, b_ac, b_vs]
+end
+
+"""
+$(SIGNATURES)
+Compute Scattering coefficient due to bubble suspended in water.
+Damping effects are included, and no assumptions about spherical nature of bubbles made.
+
+Parameters:
+- bubble 'radius' in meters
+- frequency vector 'f' in Hz
+- bubble 'depth' default: 0 m
+- water temperature 'T' in Kelvin. default 293.15K
+- atmospheric pressure 'p0' in Pa. default: 1.013e5 Pa
+- gas number. If 2: Oxygen, 3: Nitrogen, 4: Methane, any other number: Air. Default is 1.
+- soundspeed 'c' in m/s. default: 1500
+- seawater density 'ρ' in kg m^-3. default: 1022.476
+"""
+function scattering_coeff(radius::Real, f::Real; depth::Real=0, T::Real=293.15, p0::Real=1.013e5, gas::Integer=1, c::Real=1500, ρ::Real=1022.476)
+  ω = 2π*f
+
+  bubbleparams = bubble_resonance_and_damping(radius, f, z=depth, T=T, p0=p0, gas=gas, c=c)
+  ω0 = bubbleparams[1]
+  b_th = bubbleparams[2]
+  b_ac = bubbleparams[3]
+  b_vs = bubbleparams[4]
+  β = b_th .+ b_ac .+ b_vs #sum of all 3 types of damping - thermal, viscous and acoustic
+  denom = (ω0./ω).^2 .- 1 .+ 2im*β.*(1 ./ω)
+  gscat = radius./denom
+  return gscat
```

Comment:
Can drop the `return` and the variable `gscat` by simply having `radius ./ denom` on the last line.